futurerestore won't compile

________________________________________________________________________________
| ~ @ Pro-Late-2013
| => brew install futurerestore
==> Installing futurerestore from stek29/idevice
==> Cloning https://github.com/tihmstar/futurerestore.git
Cloning into '/Users/suzughia/Library/Caches/Homebrew/futurerestore--git'...
==> Checking out revision 88861b6128b7ae736bad4880e07f9026e025a537
Note: checking out '88861b6128b7ae736bad4880e07f9026e025a537'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 88861b6 Exclude subdirs from install
Submodule 'external/idevicerestore' (https://github.com/tihmstar/idevicerestore.git) registered for path 'external/idevicerestore'
Submodule 'external/tsschecker' (https://github.com/tihmstar/tsschecker.git) registered for path 'external/tsschecker'
Cloning into '/Users/suzughia/Library/Caches/Homebrew/futurerestore--git/external/idevicerestore'...
Cloning into '/Users/suzughia/Library/Caches/Homebrew/futurerestore--git/external/tsschecker'...
From https://github.com/tihmstar/idevicerestore
 * branch            c97e02e22b9971471db5dcb3b9e02eb30222d6c0 -> FETCH_HEAD
Submodule path 'external/idevicerestore': checked out 'c97e02e22b9971471db5dcb3b9e02eb30222d6c0'
Submodule path 'external/tsschecker': checked out 'b9d193aa6e6d24421094873c830692d02d8b32f5'
Submodule 'external/jssy' (https://github.com/tihmstar/jssy.git) registered for path 'external/tsschecker/external/jssy'
Cloning into '/Users/suzughia/Library/Caches/Homebrew/futurerestore--git/external/tsschecker/external/jssy'...
Submodule path 'external/tsschecker/external/jssy': checked out 'e17d3c8ec5216692efbbe59bbe9801bb7661e07d'
/Users/suzughia/Library/Caches/Homebrew/futurerestore--git/external/idevicerestore
/Users/suzughia/Library/Caches/Homebrew/futurerestore--git/external/tsschecker
/Users/suzughia/Library/Caches/Homebrew/futurerestore--git/external/tsschecker/external/jssy
Error: futurerestore 246 is already installed
To install 182, first run `brew unlink futurerestore`.
________________________________________________________________________________
| ~ @ Pro-Late-2013
| => brew unlink futurerestore
Unlinking /usr/local/Cellar/futurerestore/246... 1 symlinks removed
________________________________________________________________________________
| ~ @ Pro-Late-2013 (suzughia) 
| => brew install futurerestore
==> Installing futurerestore from stek29/idevice
==> Cloning https://github.com/tihmstar/futurerestore.git
Updating /Users/suzughia/Library/Caches/Homebrew/futurerestore--git
==> Checking out revision 88861b6128b7ae736bad4880e07f9026e025a537
HEAD is now at 88861b6 Exclude subdirs from install
HEAD is now at 88861b6 Exclude subdirs from install
Entering 'external/idevicerestore'
Entering 'external/tsschecker'
Synchronizing submodule url for 'external/jssy'
Entering 'external/tsschecker/external/jssy'
/Users/suzughia/Library/Caches/Homebrew/futurerestore--git/external/idevicerestore
/Users/suzughia/Library/Caches/Homebrew/futurerestore--git/external/tsschecker
/Users/suzughia/Library/Caches/Homebrew/futurerestore--git/external/tsschecker/external/jssy
==> ./autogen.sh --disable-debug --disable-dependency-tracking --disable-silent-rules --prefix=/usr/local/Cellar/futurerestor
==> make
Last 15 lines from /Users/suzughia/Library/Logs/Homebrew/futurerestore/02.make:
        } else if (event->type == IRECV_DEVICE_REMOVE) {
                                  ^~~~~~~~~~~~~~~~~~~
                                  IDEVICE_DEVICE_REMOVE
/usr/local/Cellar/libimobiledevice/HEAD-0584aa9_3/include/libimobiledevice/libimobiledevice.h:58:2: note: 'IDEVICE_DEVICE_REMOVE' declared here
        IDEVICE_DEVICE_REMOVE,
        ^
idevicerestore.c:274:30: error: no member named 'device_info' in 'idevice_event_t'
                if (client->ecid && event->device_info->ecid == client->ecid) {
                                    ~~~~~  ^
10 errors generated.
make[4]: *** [libidevicerestore_la-idevicerestore.lo] Error 1
make[3]: *** [all-recursive] Error 1
make[2]: *** [all] Error 2
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/stek29/homebrew-idevice/issues

________________________________________________________________________________

I, too, am unable to install `futurerestore`. However, I seem to be facing a different problem.

```
[stratosphere:~] unilock% brew install futurerestore --with-libipatcher --HEAD
==> Installing futurerestore from stek29/idevice
==> Cloning https://github.com/tihmstar/futurerestore.git
Updating /Users/unilock/Library/Caches/Homebrew/futurerestore--git
==> Checking out branch master
Already on 'master'
Your branch is up to date with 'origin/master'.
HEAD is now at 88861b6 Exclude subdirs from install
Entering 'external/idevicerestore'
Entering 'external/tsschecker'
Synchronizing submodule url for 'external/jssy'
Entering 'external/tsschecker/external/jssy'
/Users/unilock/Library/Caches/Homebrew/futurerestore--git/external/idevicerestore
/Users/unilock/Library/Caches/Homebrew/futurerestore--git/external/tsschecker
/Users/unilock/Library/Caches/Homebrew/futurerestore--git/external/tsschecker/external/jssy
Warning: A newer Command Line Tools release is available.
Update them from Software Update in System Preferences or run:
  softwareupdate --all --install --force

If that doesn't show you an update run:
  sudo rm -rf /Library/Developer/CommandLineTools
  sudo xcode-select --install

Alternatively, manually download them from:
  https://developer.apple.com/download/more/.

==> ./autogen.sh --disable-debug --disable-dependency-tracking --disable-silent-
Last 15 lines from /Users/unilock/Library/Logs/Homebrew/futurerestore/01.autogen.sh:
checking whether clang understands -c and -o together... yes
checking dependency style of clang... none
checking whether we need platform-specific build settings... checking for pkg-config... /usr/local/Homebrew/Library/Homebrew/shims/mac/super/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for libplist >= 2.0.0... no
configure: error: Package requirements (libplist >= 2.0.0) were not met:

No package 'libplist' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables libplist_CFLAGS
and libplist_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/stek29/homebrew-idevice/issues

These open issues may also help:
futurerestore won't compile https://github.com/stek29/homebrew-idevice/issues/17

Error: A newer Command Line Tools release is available.
Update them from Software Update in System Preferences or run:
  softwareupdate --all --install --force

If that doesn't show you an update run:
  sudo rm -rf /Library/Developer/CommandLineTools
  sudo xcode-select --install

Alternatively, manually download them from:
  https://developer.apple.com/download/more/.

Please create pull requests instead of asking for help on Homebrew's GitHub,
Discourse, Twitter or IRC.
[stratosphere:~] unilock%
```

I have libplist installed with `--HEAD`, so I'm not sure what the issue is.

--- a/Formula/futurerestore.rb
+++ b/Formula/futurerestore.rb
@@ -2,8 +2,8 @@ class Futurerestore < Formula
   desc "Hacked up idevicerestore wrapper"
   homepage "https://github.com/tihmstar/futurerestore"
   url "https://github.com/tihmstar/futurerestore.git",
-    :revision => "88861b6128b7ae736bad4880e07f9026e025a537"
-  version "182"
+    :revision => "2f0686d3559f1543fc0c0c741b68e1ae940c5a30"
+  version "186"
 
   head "https://github.com/tihmstar/futurerestore.git"
 

